### PR TITLE
Org login, Badge Create Endpoints

### PR DESCRIPTION
**Pull request by nikiljos**
### Updates in this PR
- Custom express error response
- Org Login endpoint
- Org auth middleware
- Badge Create endpoint

<hr>

#### Tracker Topics covered
Nothing New

#### Extra Topics covered
- [ ] Custom Express error middleware

### CODE REVIEW

File: backend/src/services/badge.service.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        title,
+        desc,
+        org
+    })
+    .then(data=>{
+        if(data){
+            resolve(data._id)
+        }
+        else{
+            reject("Unable to create")
```
Comment (aditya-190) [inline backend/src/services/badge.service.ts]: I think it should be wrapped in Error()

File: frontend/src/components/Login.jsx [hunk 2]
```diff
@@ -26,6 +26,7 @@ const Login = () => {
       })
       .catch((err) => console.log("Erorr in custom login", err));
   }
+  console.log(token)
```
Comment (aditya-190) [inline frontend/src/components/Login.jsx:29]: Typically you should remove all console.log before merging them into your main code.

**Review by aditya-190 — APPROVED** [review]
Looks good to me. Pls proceed with merging.

File: frontend/src/components/Login.jsx (hunk 2 above)
Comment (nikiljos) [reply frontend/src/components/Login.jsx:29]: Kept there for ease in API testing. Will remove before moving to prod.

File: backend/src/services/badge.service.ts (hunk 1 above)
Comment (nikiljos) [reply backend/src/services/badge.service.ts]: Done

**Comment by nikiljos** [comment]
Pls check now...

**Comment by aditya-190** [comment]
> Pls check now...

Nice Work. You can merge the changes now.

**Pull request merged**